### DISCUSSION TITLE
Sort by Date Added not matching stable if Lazer was installed previously

### DISCUSSION THREAD

**Discussion by parzypeevesy** [ppy/osu#27545] [Q&A] [answered]
I am aware of the changes made to lazer a while ago, where sorting by date added now mirrors stable. However, it's not working. I first installed lazer before that change was made, and so the date added order was totally messed up. Even after a full reinstall, the old order is still used and not the current one. I tried to delete everything on my computer related to Lazer and reinstall and it's still not working.

**Comment by SupDos** [comment, marked as answer]
You'd need to re-import from stable for it to copy the correct date added over. Not sure if it will work if the beatmaps are already in lazer - if it doesn't, you'll need to delete your beatmaps first and then import from stable
